bundles/specmate-dbprovider-h2/src/specmate/dbprovider/h2/AttributeToSQLMapper.java

```diff
@@ -97,14 +97,23 @@ public void migrateNewDateAttribute(String objectName, String attributeName, Dat
 	}
 
 	@Override
-	public void migrateNewReference(String objectName, String attributeName) throws SpecmateException {
+	public void migrateNewObjectReference(String objectName, String attributeName) throws SpecmateException {
+		migrateNewReference(objectName, attributeName, "BIGINT");
+	}
+	
+	@Override
+	public void migrateNewStringReference(String objectName, String attributeName) throws SpecmateException {
+		migrateNewReference(objectName, attributeName, "VARCHAR(32672)");
+	}
+	
+	private void migrateNewReference(String objectName, String attributeName, String type) throws SpecmateException{
 		String failmsg = "Migration: Could not add column " + attributeName + " to table " + objectName + ".";
 		String tableNameList = objectName + "_" + attributeName + "_LIST";
 		List<String> queries = new ArrayList<>();
-		queries.add("ALTER TABLE " + objectName + " ADD COLUMN " + attributeName + " INTEGER");
+		queries.add("ALTER TABLE " + objectName + " ADD COLUMN " + attributeName + " " + type);
 
```

Comment:
I'm wondering whether this change is correct as it modifies the behavior w.r.t. the previous implementation, even though it is intended to make the code more flexible (i.e. create generic references). 
Here, for the "ALTER TABLE..." query, the new column type was INTEGER when a new reference is created. Now it is BIGINT. I think it should stay INTEGER, i.e. not be parameterized, analogous to the oracle implementation. 

If this is indeed a bug, could you also check why the test cases do not capture it?